src/System.Buffers.Primitives/System/Buffers/ReadOnlyBuffer.cs

```diff
@@ -98,18 +98,6 @@ public BufferHandle Pin()
             }
         }
 
-        public bool TryGetArray(out ArraySegment<T> buffer)
```

Comment:
You can get here if _owner.TryGetArray returns false (because the buffer is not array bases).